### ISSUE TITLE
Spyder problem

### ISSUE THREAD

**Issue by MonicaSai7** [spyder-ide/spyder#6109]
## Description

**What steps will reproduce the problem?**

1. 
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "C:\Users\Monica\Anaconda3\lib\site-packages\spyder_profiler\widgets\profilergui.py", line 263, in <lambda>
    self.finished(ec, es))
  File "C:\Users\Monica\Anaconda3\lib\site-packages\spyder_profiler\widgets\profilergui.py", line 328, in finished
    self.show_data(justanalyzed=True)
  File "C:\Users\Monica\Anaconda3\lib\site-packages\spyder_profiler\widgets\profilergui.py", line 349, in show_data
    self.datatree.load_data(self.DATAPATH)
  File "C:\Users\Monica\Anaconda3\lib\site-packages\spyder_profiler\widgets\profilergui.py", line 461, in load_data
    stats_indi = [pstats.Stats(profdatafile),]
  File "C:\Users\Monica\Anaconda3\lib\pstats.py", line 72, in __init__
    self.init(arg)
  File "C:\Users\Monica\Anaconda3\lib\pstats.py", line 86, in init
    self.load_stats(arg)
  File "C:\Users\Monica\Anaconda3\lib\pstats.py", line 99, in load_stats
    with open(arg, 'rb') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\Monica\\.spyder-py3\\profiler.results'

## Version and main components

* Spyder Version: 3.2.5 
* Python Version: 3.6.3
* Qt Versions: 5.6.2, PyQt5 5.6 on Windows

## Dependencies
```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.21.1 (OK)
numpy >=1.7      :  1.13.3 (OK)
sphinx >=0.6.6   :  1.6.3 (OK)
rope >=0.9.4     :  0.10.5 (OK)
jedi >=0.9.0     :  0.10.2 (OK)
nbconvert >=4.0  :  5.3.1 (OK)
sympy >=0.7.3    :  1.1.1 (OK)
cython >=0.21    :  0.26.1 (OK)
qtconsole >=4.2.0:  4.3.1 (OK)
IPython >=4.0    :  6.1.0 (OK)
pylint >=0.25    :  1.7.4 (OK)

```

**Comment by CAM-Gerlach** [comment]
Thanks for reporting. Interesting, this appears to be a duplicate of  #5587 and thus should be fixed in Spyder 3.2.5, but according to your error report, that is the version you are running. @rlaverde , any thoughts?

In any case, you'll need to provide some more details so we can isolate the problem. Specifically, we'll need to know specifically what you were doing, step by step, when the problem occurred. Does it only happen when profiling on specific code, or anything you try? Can you share this code? Has profiling worked before, or is the first time you tried it? Since updating Spyder? Also, is there anything else relevant you can think of, and anything you've tried to fix it?

Thanks, and we look forward to hearing from you. If we don't, we'll have to close this issue after one week (7 days), as we won't be able to isolate the problem. Best of luck!

**Comment by CAM-Gerlach** [comment]
Closed due to lack of response. Please re-open if you can add the requested information, thanks.
